public/components/CsvItem/CsvItem.js

```diff
@@ -0,0 +1,175 @@
+
```

Comment:
From what I see in `main/main.js`, merging into the codebase putting `CsvItem` and `Error` under something like `components/CsvReports` is a good option